setup.py

```diff
@@ -40,6 +40,9 @@
         'platform_system=="Windows"': [
             'pypiwin32'  # TODO: specify a version number, move under install_requires
         ],
+        'websocket': [
+            "websockets==4.0.1"
```

Comment:
cc @carver 

I think this should be added as a core requirement that is installed by default.

Reply:
will do

Reply:
:+1: core requirement seems fine here